Tests/SonarScanner.MSBuild.Common.Test/ProcessRunnerTests.cs

```
@@ -356,6 +358,20 @@ public void ProcRunner_MissingExe()
         context.Logger.Should().HaveErrorOnce("Execution failed. The specified executable does not exist: missingExe.foo");
     }
 
+    [TestMethod]
+    public void ProcRunner_MissingExe_ExeMustExists_False()
+    {
+        var context = new ProcessRunnerContext(TestContext, string.Empty)
+        {
+            ProcessArgs = new ProcessRunnerArguments("missingExe.foo", false) { ExeMustExists = false }
+        };
+
+        FluentActions.Invoking(context.Execute).Should().Throw<Win32Exception>().Which.Message.Should().BeOneOf(
```

This exception is propagated from the SonarEngineWrapper to the PostProcessor, to the Bootstrapper, to Program.Main which terminates with this exception. I think we should catch this exception when the SonarEngineWrapper is called and transform itto our own AnalysisException which is then caught and logged by the bootstrapper:
https://github.com/SonarSource/sonar-scanner-msbuild/blob/d8d66d128f9a98284ada4e70c6814654d19957d6/src/SonarScanner.MSBuild/BootstrapperClass.cs?plain=1#L79C1-L79C37

---

Yes, we should catch all exceptions that are thrown by `ProcessRunner` and at least wrap them.